examples/core/graphsage/node_classification.py

```diff
@@ -0,0 +1,132 @@
+"""
+This script trains and tests a GraphSAGE model based on the information of 
+a full graph.
+
+This flowchart describes the main functional sequence of the provided example.
+main
+│
+├───> Load and preprocess full dataset
+│
+├───> Instantiate SAGE model
+│
+├───> train
+│     │
+│     └───> Training loop
+│           │
+│           └───> SAGE.forward
+└───> test
+      │
+      └───> Evaluate the model
+"""
+import argparse
+
+import dgl.nn as dglnn
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl import AddSelfLoop
+from dgl.data import CiteseerGraphDataset, CoraGraphDataset, PubmedGraphDataset
+
+
+class SAGE(nn.Module):
+    def __init__(self, in_size, hid_size, out_size):
+        super().__init__()
+        self.layers = nn.ModuleList()
+        # two-layer GraphSAGE-mean
```

Comment:
For comment, capitalize the first char and end with dot.
```suggestion
        # Two-layer GraphSAGE-mean.
```